src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionCoordinatorModel.swift

```diff
@@ -243,5 +243,32 @@ class ExposureSubmissionCoordinatorModel {
 			fatalError("Cannot set submission consent, no corona test type is set")
 		}
 	}
+	
+	func createDiaryTestEntry() {
```

Comment:
There is a lot of coronaTestService access in this func. The logic seems to fit better into coronaTestService.